qrand.go

```diff
@@ -66,6 +67,29 @@ func Read(buf []byte) (n int, err error) {
 	return Reader.Read(buf)
 }
 
+// Source represents an alternative source that can also generate uniformly-distributed pseudo-random values
+type Source struct{}
```

Comment:
We can make this unexported now, can't we?

Reply:
By the way, it doesn't generate "pseudo-random" values; they're _actually_ random!